testcases/multizone_residential_hydronic/config.py

```diff
@@ -25,6 +26,7 @@ def get_config():
     # Enter configuration information
     'name'     : 'multizone_residential_hydronic',
     'fmupath'  : 'models/wrapped.fmu',
+    'area'     : 8.3*14.4,
```

Comment:
Shouldn't we remove the area of the garage here to use only the area of conditioned zones? It'd be: `8.3*14.4-20.52` I think it's more fair since thermal discomfort and indoor air quality are also computed based on those zones only.

Reply:
Also I'm double checking the lengths of the building and always get a length of 12.7m and a width of 8m. This is based on the specification of the lengths for each zone and orientation at the beginning of the model. So if I'm not mistaken I think it should actually be `8*12.7-20.52`. I'm not sure where the 8.3m and 14.4m are coming from, I'm not able to reproduce those values again. I think it'd be better if we update the figure of the documentation accordingly, I can open a new issue for that.

Reply:
> Shouldn't we remove the area of the garage here to use only the area of conditioned zones? It'd be: 8.3*14.4-20.52 I think it's more fair since thermal discomfort and indoor air quality are also computed based on those zones only.

You bring up a couple of clarifying points.  I would suggest that the floor area used be total floor area, not only conditioned floor area, since this is what is used in [EnergyStar](https://www.energystar.gov/buildings/benchmark/understand_metrics/what_eui) and [CBECS](https://www.eia.gov/consumption/commercial/data/2012/c&e/cfm/c1.php) EUI reporting.  The question remains to me though if a garage should be considered building floor area?  I'd say probably not, since a garage could be detached, or it may not be included within the insulated envelope of the building.

> Also I'm double checking the lengths of the building and always get a length of 12.7m and a width of 8m. This is based on the specification of the lengths for each zone and orientation at the beginning of the model. So if I'm not mistaken I think it should actually be 8*12.7-20.52. I'm not sure where the 8.3m and 14.4m are coming from, I'm not able to reproduce those values again. I think it'd be better if we update the figure of the documentation accordingly, I can open a new issue for that.

I took the numbers 8.3*14.4 from the schematic in the documentation and did not check the actual modelica model.  They also match the text in the documentation that states the floor area is approximately 120 m^2.  Both should match what is actually in the Modelica model, and specifying what the floor area is of the garage and whether or not it is included in the area calculation for EUI, cost, emissions would be good.

Reply:
I suggest to merge https://github.com/ibpsa/project1-boptest/compare/master...JavierArroyoBastida:issue333_correctSchema?expand=1 first, and use `12.7*8-20.52` as we don't consider garage as part of the floor area. Notice though that if the attic is to be considered floor area and we decide to inclue unconditioned zones then the attic floor should count as well and the area to be used would be: `12.7*8*2-20.52`. I still think it makes more sense to account only for conditioned spaces, but I guess it's a bit subjective like our discussion in https://github.com/ibpsa/project1-boptest/issues/237 and I don't have a strong opinion on it so @dhblum feel free to decide on what's best.

Reply:
Per my suggestion [here](https://github.com/ibpsa/project1-boptest/issues/237#issuecomment-856822390), the unconditioned attic and unconditioned garage should NOT be included in the floor area.  Please make sure changes in #334 are consistent with this.  I would specify in the documentation something like, "the area not including the unconditioned attic and unconditioned garage is <area>."

Reply:
Sounds good. I've stated that in https://github.com/ibpsa/project1-boptest/pull/334/commits/62336503a523e440042500a803efb5156cf476dd.

Reply:
Cool thanks.  I've updated the area in ``config.py`` and unit test reference results in https://github.com/ibpsa/project1-boptest/pull/330/commits/54ea93b93546d41f448c34e95ada7e187d7b5422.

Reply:
#334 has been merged and this branch updated with latest master.